source/JsMaterialX/test/codeExamples.spec.js

```diff
@@ -107,7 +107,7 @@ describe('Code Examples', () => {
             }
         }
         expect(elementCount).to.equal(21);
-        expect(nodeCount).to.equal(5);
+        expect(nodeCount).to.equal(1);
```

Comment:
It was passing before, because we used another check in the loop (`instanceof mx.Node` vs `isANode('image')`). So now we're only looking at image nodes explicitly, whereas before all nodes where counted. That's also the answer the other question: The parameter of `isANode` is the node type to check for, e.g. image.